test/end-to-end/Classes/declarations.js

```diff
@@ -0,0 +1,27 @@
+-
+class Point {
+    public x;
+    public y;
+    protected square;
+    private name;
+    
+    constructor(x, y) {
+        this.x = x;
+        this.y = y;
+        this.square = x * y;
+        this.name = "";
+    }
+}
+
+class Measurements<T> {
```

Comment:
Javascript doesn't have templating or interfaces. These should end up blank, like in Python and Ruby.